api/proto/teleport/autoupdate/v1/autoupdate.proto

```diff
@@ -33,8 +33,25 @@ message AutoUpdateConfig {
 
 // AutoUpdateConfigSpec encodes the parameters of the autoupdate config object.
 message AutoUpdateConfigSpec {
-  // ToolsAutoupdate encodes the feature flag to enable/disable tools autoupdates.
-  bool tools_autoupdate = 1;
+  reserved 1;
+  reserved "tools_autoupdate"; // ToolsAutoupdate is replaced by tools.mode.
+  AutoUpdateConfigTools tools = 2;
+}
+
+// AutoUpdateConfigTools encodes the parameters for client tools auto updates.
+message AutoUpdateConfigTools {
+  // Mode defines state of the client tools auto update.
+  ToolsUpdateMode mode = 1;
+}
```

Comment:
Should this message be named `AutoUpdateConficSpecTools`? That's what we put in RFD 184 but I cvan change the RFD if needed.

Reply:
since it already in rfd, changing this one, and for version